code/game/objects/structures/dropship_ammo.dm

```diff
@@ -494,12 +494,12 @@
 
 //minirockets
 
-/obj/structure/ship_ammo/minirocket
+/obj/structure/ship_ammo/cas/minirocket
 	name = "mini rocket stack"
 	desc = "A pack of explosive laser guided mini rockets. Moving this will require some sort of lifter."
 	icon_state = "minirocket"
 	icon = 'icons/Marine/mainship_props.dmi'
-	equipment_type = /obj/structure/dropship_equipment/weapon/minirocket_pod
+	equipment_type = /obj/structure//obj/structure/dropship_equipment/cas/weapon/minirocket_pod
```

Comment:
fucked up here

Reply:
Why does it need a second obj/structure?

Reply:
Hold on, one sec l, misread this

Reply:
Should be fixed now